app/interactors/rake_interactor.rb

```diff
@@ -18,6 +18,8 @@ def call
     $stdout = log_buffer
 
     begin
+      Rake.application.init
+      Rake.application.load_rakefile
```

Comment:
We instead load the rake tasks here ONLY when we call rake task thru rake interactor instead of thru global application.rb